BloodManagmentSystem/Content/Site.css

```diff
@@ -1,24 +1,67 @@
 ﻿body {
-    padding-top: 50px;
-    padding-bottom: 20px;
+  padding-top: 50px;
+  padding-bottom: 20px;
+  font-family: 'Titillium Web', Sans-Serif;
+  font-size: 16px;
 }
-
 /* Set padding to keep content from hitting the edges */
 .body-content {
```

Comment:
There's also built-in bootstrap class `.container-fluid` with similar paddings from what I remember

Reply:
This was default style override in ASP.NET MVC project